### PR TITLE
Document Promptl Version

### PR DESCRIPTION
**Pull request by csansoon**
Done 3 simple things:
- Added a `promptlVersion` attribute to documents. `0` by default, which represents the old version.
- Removed `promptl` package directory, as it is no longer in the monorepo
- Installed `@latitude-data/promptl` in `web`, `gateway` and `core` packages.

### CODE REVIEW

File: packages/core/src/schema/models/documentVersions.ts [hunk 1]
```diff
@@ -23,6 +24,7 @@ export const documentVersions = latitudeSchema.table(
     content: text('content').notNull().default(''),
     resolvedContent: text('resolved_content'),
     contentHash: text('content_hash'),
+    promptlVersion: integer('promptl_version').notNull().default(0),
```
Comment (geclos) [inline packages/core/src/schema/models/documentVersions.ts:27]: move drizzle changes to another PR and deploy that before merging this

Comment (csansoon) [reply packages/core/src/schema/models/documentVersions.ts:27]: https://github.com/latitude-dev/latitude-llm/pull/629

File: packages/core/package.json [hunk 2]
```diff
@@ -51,19 +51,25 @@
...
     "@types/node": "^22.5.0",
+    "@types/nodemailer": "^6.4.15",
+    "@types/nodemailer-html-to-text": "^3.1.3",
+    "@types/nodemailer-mailgun-transport": "^1.4.6",
     "@types/pg": "^8.11.6",
+    "@types/react": "18.3.0",
+    "@types/react-dom": "18.3.0",
     "@types/uuid": "^10.0.0",
     "ai": "^3.4.7",
-    "loops": "^3.3.0",
```
Comment (csansoon) [inline packages/core/package.json:66]: prettier haciendo de las suyas

**Review by geclos — APPROVED** [review]
